.github/workflows/conformance-externalworkloads-v1.10.yml

```diff
@@ -0,0 +1,341 @@
+name: External workloads (ci-external-workloads-v1.10)
```

Comment:
What's this file about?

Reply:
Testing for backport branches via `test-backport-1.10`, see #16944 :)

Reply:
:scream: :scream: :scream:

Reply:
Is it too late to ask to go back to Jenkins?

Reply:
I'll be working on removing that horrible thing ASAP 😅